### PR TITLE
doc/md: add clickhouse

### PR DESCRIPTION
**Pull request by datdao**

### CODE REVIEW

File: doc/md/concepts/dev.mdx [hunk 1]
```diff
@@ -108,6 +108,20 @@ docker+postgres://org/image:tag
 --dev-url "sqlite://dev?mode=memory"
 ```
 
+</TabItem>
+<TabItem value="clickhouse" label="ClickHouse">
+
+Atlas does not support initializing a ClickHouse devDB through `docker://` at the momment, but you can use create a local devDB by
+running the following command:
+
```
Comment (a8m) [inline doc/md/concepts/dev.mdx:116]:
```suggestion
Atlas does not support yet initializing a ClickHouse dev-database through `docker://`, but you can create a local container by running the following command:
```

File: doc/md/concepts/dev.mdx [hunk 2]
```diff
@@ -108,6 +108,20 @@ docker+postgres://org/image:tag
...
+```shell
+docker run -d --name clickhouse-dev -p 9000:9000 -d clickhouse/clickhouse-server:latest
```
Comment (a8m) [inline doc/md/concepts/dev.mdx:118]:
```suggestion
docker run -d --rm --name clickhouse-dev -p 9000:9000 -d clickhouse/clickhouse-server:latest
```

File: doc/md/declarative/diff.mdx [hunk 3]
```diff
@@ -174,6 +185,17 @@ atlas schema diff \
   --to "postgres://postgres:pass@remote:5432/example?search_path=public"
 ```
 
+</TabItem>
+<TabItem value="clickhouse">
+
+Compare two ClickHouse schemas/databases named example:
```
Comment (a8m) [inline doc/md/declarative/diff.mdx:191]:
```suggestion
Compare two ClickHouse schemas/named-databases:
```

File: doc/md/declarative/diff.mdx [hunk 4]
```diff
@@ -607,6 +685,45 @@ atlas schema diff \
...
+
+data "external_schema" "sequelize" {
+  program = [
+    "npx",
+    "@ariga/atlas-provider-sequelize",
+    "load",
+    "--path", "./path/to/models",
+    "--dialect", "clickhouse",
+  ]
+}
```
Comment (a8m) [inline doc/md/declarative/diff.mdx:710]: I don't think we support this

**Review by a8m — COMMENTED** [review]
🚀

**Review by giautm — APPROVED** [review]

**Pull request merged**